flask/templates/Friendspage.html

```diff
@@ -0,0 +1,247 @@
+
+<html>
+    <head> 
+
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1 ">
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
+<script src="https://kit.fontawesome.com/80c51aa779.js" crossorigin="anonymous"></script>
+
+</head>
+
```

Comment:
Try to take out the head stuff (bootstrap, fontawesome, etc) to the base.html file and use the {% extend %} thing to get everything from base.html in the friends page.

Reply:
check to see if this breaks the other pages though.